tests/Microsoft.Identity.Test.Common/Core/Mocks/MockHelpers.cs

```diff
@@ -116,6 +116,14 @@ public static string GetMsiImdsSuccessfulResponse()
           "\"ext_expires_in\":\"12345\",\"token_type\":\"Bearer\"}";
         }
 
+        public static string GetMsiSuccessfulWithExpiresOnResponse(int hoursToAdd)

```

Comment:
You can reuse https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/blob/main/tests/Microsoft.Identity.Test.Common/Core/Mocks/MockHelpers.cs#L102. here.